### PR TITLE
Fix intermittent failure of messaging tests

### LINKED ISSUE #1023: Intermittent failure of messaging tests
Seems to be happening since the Quarkus 1.3.1.Final upgrade:

```
2020-04-02T15:40:32.5735131Z [INFO] -------------------------------------------------------
2020-04-02T15:40:32.5736596Z [INFO]  T E S T S
2020-04-02T15:40:32.5738785Z [INFO] -------------------------------------------------------
2020-04-02T15:40:33.2615346Z [INFO] Running org.apache.camel.quarkus.component.messaging.it.JmsTest
2020-04-02T15:40:45.9624920Z 2020-04-02 15:40:37,298 INFO  [org.apa.cam.qua.com.mes.it.ActiveMQTestResource] (main) TestcontainersConfiguration(environmentProperties={docker.client.strategy=org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy}, properties={docker.client.strategy=org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy, pull.pause.timeout=120})
2020-04-02T15:40:45.9626560Z 2020-04-02 15:40:37,334 INFO  [org.tes.doc.DockerClientProviderStrategy] (main) Loaded org.testcontainers.dockerclient.EnvironmentAndSystemPropertyClientProviderStrategy from ~/.testcontainers.properties, will try it first
2020-04-02T15:40:45.9627304Z 2020-04-02 15:40:37,945 INFO  [org.tes.doc.EnvironmentAndSystemPropertyClientProviderStrategy] (main) Found docker client settings from environment
2020-04-02T15:40:45.9628159Z 2020-04-02 15:40:37,945 INFO  [org.tes.doc.DockerClientProviderStrategy] (main) Found Docker environment with Environment variables, system properties and defaults. Resolved dockerHost=unix:///var/run/docker.sock
2020-04-02T15:40:45.9628776Z 2020-04-02 15:40:38,120 INFO  [org.tes.DockerClientFactory] (main) Docker host IP address is localhost
2020-04-02T15:40:45.9629355Z 2020-04-02 15:40:38,145 INFO  [org.tes.DockerClientFactory] (main) Connected to docker: 
2020-04-02T15:40:45.9629729Z   Server Version: 3.0.11+azure
2020-04-02T15:40:45.9630071Z   API Version: 1.40
2020-04-02T15:40:45.9630374Z   Operating System: Ubuntu 18.04.4 LTS
2020-04-02T15:40:45.9631078Z   Total Memory: 6945 MB
2020-04-02T15:40:45.9634922Z 2020-04-02 15:40:38,179 WARN  [org.tes.uti.RegistryAuthLocator] (main) Failure when attempting to lookup auth config (dockerImageName: quay.io/testcontainers/ryuk:0.2.3, configFile: /home/runner/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/runner/.docker/config.json (No such file or directory)
2020-04-02T15:40:45.9637795Z 2020-04-02 15:40:38,755 INFO  [org.tes.DockerClientFactory] (main) Ryuk started - will monitor and terminate Testcontainers containers on JVM exit
2020-04-02T15:40:45.9781698Z 2020-04-02 15:40:38,755 INFO  [org.tes.DockerClientFactory] (main) Checking the system...
2020-04-02T15:40:45.9782562Z 2020-04-02 15:40:38,756 INFO  [org.tes.DockerClientFactory] (main) ?? Docker server version should be at least 1.6.0
2020-04-02T15:40:45.9783476Z 2020-04-02 15:40:38,844 INFO  [org.tes.DockerClientFactory] (main) ?? Docker environment should have more than 2GB free disk space
2020-04-02T15:40:45.9784165Z 2020-04-02 15:40:38,865 INFO  [doc.11.0-alpine]] (main) Creating container for image: vromero/activemq-artemis:2.11.0-alpine
2020-04-02T15:40:45.9784992Z 2020-04-02 15:40:38,865 WARN  [org.tes.uti.RegistryAuthLocator] (main) Failure when attempting to lookup auth config (dockerImageName: vromero/activemq-artemis:2.11.0-alpine, configFile: /home/runner/.docker/config.json. Falling back to docker-java default behaviour. Exception message: /home/runner/.docker/config.json (No such file or directory)
2020-04-02T15:40:45.9785623Z 2020-04-02 15:40:38,951 INFO  [doc.11.0-alpine]] (main) Starting container with ID: b7d95f02d472b815fd49db0027ef451833c0d7699e2da685b34a737ae84dc849
2020-04-02T15:40:45.9786284Z 2020-04-02 15:40:39,260 INFO  [doc.11.0-alpine]] (main) Container vromero/activemq-artemis:2.11.0-alpine is starting: b7d95f02d472b815fd49db0027ef451833c0d7699e2da685b34a737ae84dc849
2020-04-02T15:40:45.9786835Z 2020-04-02 15:40:45,056 INFO  [doc.11.0-alpine]] (main) Container vromero/activemq-artemis:2.11.0-alpine started in PT7.733S
2020-04-02T15:40:46.4072427Z 2020-04-02 15:40:46,406 INFO  [org.apa.cam.mai.BaseMainSupport] (main) Auto-configuration summary:
2020-04-02T15:40:46.4077196Z 2020-04-02 15:40:46,407 INFO  [org.apa.cam.mai.BaseMainSupport] (main) 	camel.component.paho.brokerurl=tcp://127.0.0.1:32796
2020-04-02T15:40:46.4085294Z 2020-04-02 15:40:46,407 INFO  [org.apa.cam.mai.BaseMainSupport] (main) 	camel.component.paho.configuration.username=artemis
2020-04-02T15:40:46.4096727Z 2020-04-02 15:40:46,408 INFO  [org.apa.cam.mai.BaseMainSupport] (main) 	camel.component.paho.configuration.password=simetraehcapa
2020-04-02T15:40:46.4116195Z 2020-04-02 15:40:46,411 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 3.1.0 (CamelContext: camel-1) is starting
2020-04-02T15:40:46.4131731Z 2020-04-02 15:40:46,412 INFO  [org.apa.cam.imp.eng.DefaultManagementStrategy] (main) JMX is disabled
2020-04-02T15:40:46.4160320Z 2020-04-02 15:40:46,415 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) StreamCaching is not in use. If using streams then its recommended to enable stream caching. See more details at http://camel.apache.org/stream-caching.html
2020-04-02T15:40:46.4169696Z 2020-04-02 15:40:46,416 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Total 0 routes, of which 0 are started
2020-04-02T15:40:46.4181714Z 2020-04-02 15:40:46,417 INFO  [org.apa.cam.imp.eng.AbstractCamelContext] (main) Apache Camel 3.1.0 (CamelContext: camel-1) started in 0.005 seconds
2020-04-02T15:40:46.4196133Z 2020-04-02 15:40:46,418 INFO  [io.quarkus] (main) Quarkus 1.3.1.Final started in 10.922s. Listening on: http://0.0.0.0:43111
2020-04-02T15:40:46.4198362Z 2020-04-02 15:40:46,419 INFO  [io.quarkus] (main) Profile test activated. 
2020-04-02T15:40:46.4204251Z 2020-04-02 15:40:46,419 INFO  [io.quarkus] (main) Installed features: [artemis-jms, camel-core, camel-jms, camel-paho, camel-policy, camel-sjms, camel-sjms2, camel-support-common, camel-support-commons-logging, camel-support-spring, cdi, resteasy]
2020-04-02T15:40:47.7347592Z 2020-04-02 15:40:47,732 INFO  [org.apa.cam.mai.BaseMainSupport] (executor-thread-1) Autowired property: connectionFactory on component: JmsComponent as exactly one instance of type: javax.jms.ConnectionFactory found in the registry
2020-04-02T15:40:49.4391724Z 2020-04-02 15:40:49,426 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-1) HTTP Request to /messaging/paho/paho-test-queue failed, error id: 392bad98-f1eb-4bfd-81f0-b4f00435cf46-1: org.jboss.resteasy.spi.UnhandledException: org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[]
2020-04-02T15:40:49.4414553Z 	at org.jboss.resteasy.core.ExceptionHandler.handleApplicationException(ExceptionHandler.java:106)
2020-04-02T15:40:49.4415542Z 	at org.jboss.resteasy.core.ExceptionHandler.handleException(ExceptionHandler.java:372)
2020-04-02T15:40:49.4415783Z 	at org.jboss.resteasy.core.SynchronousDispatcher.writeException(SynchronousDispatcher.java:216)
2020-04-02T15:40:49.4415964Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:515)
2020-04-02T15:40:49.4416205Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:259)
2020-04-02T15:40:49.4416373Z 	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:160)
2020-04-02T15:40:49.4416549Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:362)
2020-04-02T15:40:49.4416717Z 	at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:163)
2020-04-02T15:40:49.4416881Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:245)
2020-04-02T15:40:49.4417046Z 	at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:73)
2020-04-02T15:40:49.4417210Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:122)
2020-04-02T15:40:49.4487458Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.access$000(VertxRequestHandler.java:36)
2020-04-02T15:40:49.4510658Z 	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:87)
2020-04-02T15:40:49.4511112Z 	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
2020-04-02T15:40:49.4511420Z 	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:2027)
2020-04-02T15:40:49.4511706Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1551)
2020-04-02T15:40:49.4511992Z 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1442)
2020-04-02T15:40:49.4512340Z 	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
2020-04-02T15:40:49.4512644Z 	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
2020-04-02T15:40:49.4512936Z 	at java.lang.Thread.run(Thread.java:748)
2020-04-02T15:40:49.4513208Z 	at org.jboss.threads.JBossThread.run(JBossThread.java:479)
2020-04-02T15:40:49.4513492Z Caused by: org.apache.camel.CamelExecutionException: Exception occurred during execution on the exchange: Exchange[]
2020-04-02T15:40:49.4513784Z 	at org.apache.camel.CamelExecutionException.wrapCamelExecutionException(CamelExecutionException.java:47)
2020-04-02T15:40:49.4514062Z 	at org.apache.camel.support.ExchangeHelper.extractResultBody(ExchangeHelper.java:636)
2020-04-02T15:40:49.4514361Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.extractResultBody(DefaultProducerTemplate.java:568)
2020-04-02T15:40:49.4514649Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.extractResultBody(DefaultProducerTemplate.java:564)
2020-04-02T15:40:49.4514939Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.sendBody(DefaultProducerTemplate.java:189)
2020-04-02T15:40:49.4515222Z 	at org.apache.camel.impl.engine.DefaultProducerTemplate.sendBody(DefaultProducerTemplate.java:195)
2020-04-02T15:40:49.4515820Z 	at org.apache.camel.quarkus.component.messaging.it.JmsResource.producePahoMessage(JmsResource.java:81)
2020-04-02T15:40:49.4516087Z 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-04-02T15:40:49.4516359Z 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-04-02T15:40:49.4516638Z 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-04-02T15:40:49.4516904Z 	at java.lang.reflect.Method.invoke(Method.java:498)
2020-04-02T15:40:49.4517167Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:167)
2020-04-02T15:40:49.4517438Z 	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:130)
2020-04-02T15:40:49.4517805Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.internalInvokeOnTarget(ResourceMethodInvoker.java:621)
2020-04-02T15:40:49.4518119Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTargetAfterFilter(ResourceMethodInvoker.java:487)
2020-04-02T15:40:49.4518411Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.lambda$invokeOnTarget$2(ResourceMethodInvoker.java:437)
2020-04-02T15:40:49.4518703Z 	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:362)
2020-04-02T15:40:49.4518984Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:439)
2020-04-02T15:40:49.4519263Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:400)
2020-04-02T15:40:49.4519597Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:374)
2020-04-02T15:40:49.4519871Z 	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:67)
2020-04-02T15:40:49.4520144Z 	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:488)
2020-04-02T15:40:49.4520400Z 	... 17 more
2020-04-02T15:40:49.4521226Z Caused by: Connection lost (32109) - java.io.EOFException
2020-04-02T15:40:49.4521524Z 	at org.eclipse.paho.client.mqttv3.internal.CommsReceiver.run(CommsReceiver.java:190)
2020-04-02T15:40:49.4521783Z 	at java.lang.Thread.run(Thread.java:748)
2020-04-02T15:40:49.4522032Z Caused by: java.io.EOFException
2020-04-02T15:40:49.4522289Z 	at java.io.DataInputStream.readByte(DataInputStream.java:267)
2020-04-02T15:40:49.4522606Z 	at org.eclipse.paho.client.mqttv3.internal.wire.MqttInputStream.readMqttWireMessage(MqttInputStream.java:92)
2020-04-02T15:40:49.4522888Z 	at org.eclipse.paho.client.mqttv3.internal.CommsReceiver.run(CommsReceiver.java:137)
2020-04-02T15:40:49.4523142Z 	... 1 more
2020-04-02T15:40:49.4523319Z 
2020-04-02T15:40:49.5726272Z 2020-04-02 15:40:49,571 INFO  [org.apa.cam.mai.BaseMainSupport] (executor-thread-1) Autowired property: connectionFactory on component: Sjms2Component as exactly one instance of type: javax.jms.ConnectionFactory found in the registry
2020-04-02T15:40:59.7334357Z 2020-04-02 15:40:59,732 WARN  [org.apa.act.art.cor.client] (executor-thread-1) AMQ212054: Destination address=sjms-test-queue is blocked. If the system is configured to block make sure you consume messages on this configuration.
```

### PR DESCRIPTION
**Pull request by jamesnetherton**
Fixes #1023

### CODE REVIEW

File: integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java [hunk 1]
```diff
@@ -43,6 +45,13 @@
         try {
             container = new GenericContainer(ACTIVEMQ_IMAGE)
                     .withExposedPorts(ACTIVEMQ_PORT)
+                    .withLogConsumer(new Consumer<OutputFrame>() {
+                        @Override
+                        public void accept(OutputFrame outputFrame) {
+                            System.out.println(outputFrame.getUtf8String());
```
Comment (lburgazzoli) [inline integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java:51]: Maybe this should be replaced by slf4j ?

Comment (jamesnetherton) [reply integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java:51]: I can remove the logging altogether if you'd prefer. It's just useful for debugging if we hit issues.

Comment (lburgazzoli) [reply integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java:51]: I meant [this one](https://github.com/testcontainers/testcontainers-java/blob/master/core/src/main/java/org/testcontainers/containers/output/Slf4jLogConsumer.java) that should properly handle the output frame.

Comment (lburgazzoli) [reply integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java:51]: > I can remove the logging altogether if you'd prefer. It's just useful for debugging if we hit issues.

No worries we can keep it, was just wondering if we can make it gogin to the logger so we can control it from properties, but not a huge issue

Comment (jamesnetherton) [reply integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java]: Latest commit uses `Slf4jLogConsumer` :+1:

Comment (lburgazzoli) [reply integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java]: does it work as advertised ?

Comment (jamesnetherton) [reply integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java]: Works ok. Everything gets prefixed with `STDOUT:`, so at least it's clear what's coming from the container and what isn't.

Comment (lburgazzoli) [reply integration-tests/messaging/src/test/java/org/apache/camel/quarkus/component/messaging/it/ActiveMQTestResource.java]: Ah yeah, that issue ... I think I did implement my own version some time ago, maybe I'll copy over this repo, if only I could remember where it it

**Pull request merged**